### PR TITLE
pages-data data container

### PR DESCRIPTION
**Pull request by mbland**
Depends on #4. Adds `pages-sites/Dockerfile` to produce a data container for 18F Pages sites.

This enables the `pages` container to write to `/usr/local/18f/pages/sites`, and the `nginx-18f` container to only read from it.

You can try this locally via:

``` sh
$ ./go run_data_containers

# In one shell:
$ ./go run_pages

# In another shell:
$ ./go run_nginx
```

In the Pages shell, you can freely write to files in `/usr/local/18f/pages/sites`, and you can read them in the Nginx shell, but not modify anything.

cc: @jcscottiii @ccostino @afeld

### CODE REVIEW

**Comment by mbland** [comment]
Yeah, so I cracked the nut of how to get the `pages` image to do a `s3 sync` on startup. It's pretty _sweeet_. Now I just need to implement 18F/pages-server#41, which shouldn't be terribly difficult.

**Comment by mbland** [comment]
OK, rebased this onto `master` after #4, so this is ready for review.

File: go [hunk 1]
```diff
@@ -44,63 +44,78 @@ IMAGES = %w(
...
+    puts "#{marker}\n#{message}\n#{marker}"
+    exec_cmd "docker build -t #{image} -f ./#{image}/Dockerfile ./#{image}"
+  end
 end
 
-def _remove_container(image_name)
-  exec_cmd "if $(docker ps -a | grep -q ' #{image_name}$'); then " \
-    "docker rm #{image_name}; fi"
+def_command :create_data_containers, 'Create Docker data containers' do |args|
+  (args.empty? ? DATA_CONTAINERS.keys : args).each do |container_name|
```
Comment (jcscottiii) [inline go]: if someone specifies `args` and it's not the ones specified in `DATA_CONTAINERS` we might get some weird  stuff such as an empty base_image name

Comment (mbland) [inline go]: Good catch. I'll push a commit to do some args checking in a few minutes.

**Comment by mbland** [comment]
@jcscottiii OK, I beefed up the `./go` script to avoid errors. PTAL?

**Pull request merged**